nodejs/week3/Mealsharing/src/backend/api/meals.js

```diff
@@ -0,0 +1,202 @@
+const knex = require("../database");
+const express = require("express");
+const router = express.Router();
+//get method
+router.get("/meals", async (req, res) => {
```

Comment:
If you map the router to `/meals` when you register it in `index.js` you only need to specify the rest of the path here (e.g. `/`, `/:id`, etc).